src/plugins/intel_gna/layers/gna_layer_info.hpp

```diff
@@ -358,7 +358,7 @@ class LayerInfo {
     bool isCrop() const noexcept {
         return isOfType("crop");
     }
-    bool isCropAffined() const noexcept {
+    bool isCropAffined() const {
```

Comment:
why noexcept is removed?

Reply:
> why noexcept is removed? all other methods are noexcept

It seems it was fix of coverity issue for this PR https://github.com/openvinotoolkit/openvino/pull/11373/commits/2bc971942e572fdb215bdff80a1a7fbf9ddb9cc0

Reply:
I that in body of isCropAffined() there is used GetCropParams() which may throw.
Never the less isCropAffined() is used in 

```
    bool isWeightableIdentity() const noexcept {
        return isConcatAlignFilter() || isSyntheticScaleShift() || isCropAffined();
    }
```
which means that isWeightableIdentity may throw  even if it is declared as noexcept - you will have the same celerity warning but for another method.

Reply:
@marcinkusm I removed this line from PR to avoid half measures) Anyway fixing coverity issue is separate task:
![image](https://user-images.githubusercontent.com/5219209/179746326-fbb50f12-0289-445b-a579-1f1388c603d8.png)